vm/devices/storage/disk_nvme/nvme_driver/src/driver.rs

```diff
@@ -565,7 +566,7 @@ impl<T: DeviceBacking> NvmeDriver<T> {
 
         // It is expected the device to be alive when restoring.
         if !bar0.csts().rdy() {
-            anyhow::bail!("device is gone");
+            anyhow::bail!("device is gone, csts: {:x}", u32::from(bar0.csts()));
```

Comment:
Don't re-read csts, the value could be different on the next read and that could create confusion.